_rules/non-visual-location-reference-825ule.md

```diff
@@ -0,0 +1,506 @@
+---
+id: 825ule
+name: Non-visual location reference alternative
+rule_type: atomic
+
+description: |
+  This rule checks that when there is a references to a visual location of content, there are also non-visual indicators of the location.
+
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+  wcag20:1.3.3: # Sensory Characteristics (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+  wcag-technique:G96: # Providing textual identification of items that otherwise rely only on sensory information to be understood.
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  -  (DOM Tree, CSS Styling, Accessibility Tree, etc.,)
+
+authors:
+  -  Brian Bors
+  -  Daniël strik
+  -  Wilco Fiers
+---
+
+## Applicability
+
+Any text node that includes one of the [visual reference words](), that is [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree)
+
+## Expectation
+
+Each target that describes any [web content](https://www.w3.org/TR/WCAG21/#dfn-content) through the use of the [visual reference words](), is included in a [section of the content](#section-of-content) that also describes that [web content](https://www.w3.org/TR/WCAG21/#dfn-content) by a non-visual property, except if the target is not part of an instruction about [web content](https://www.w3.org/TR/WCAG21/#dfn-content) or the visual reference word is included in the described content.
```

Comment:
Puttting the second "except" in the applicability. Would transform this rule into a completely manual rule (because the applicability would not be programmable then). This is the reason this "except" clause is in the applicability.

Reply:
I agree. And this is a very good reason to choose the first case I mentioned in https://github.com/act-rules/act-rules.github.io/pull/832#discussion_r332446248